src/js/percircle.js

```diff
@@ -123,38 +123,67 @@
                 }, 1000);
             } else if(perdown) {
                 var secs = percircle.attr('data-secs') || options.secs;
-				var timeUpText = percircle.attr('data-timeUpText') || options.timeUpText;
-				if (timeUpText.length > 8) timeUpText='the end';
-                
-				var counter=setInterval(timer, 1000); 
+                var timeUpText = percircle.attr('data-timeUpText') || options.timeUpText;
+                var reset = percircle[0].hasAttribute('data-reset') || options.reset;
```

Comment:
`.attr()` gets the value of `data-reset`, in which case there is none (meaning it is undefined).  Reset is a boolean so if you don't want to make the perdown element reset, simply don't put the attribute in the first place.